lms/envs/common.py

```diff
@@ -212,6 +212,9 @@
     # Enable flow for payments for course registration (DIFFERENT from verified student flow)
     'ENABLE_PAID_COURSE_REGISTRATION': False,
 
+    # Enable the display of cosmetic course price display (set in course advanced settings)
+    'ENABLE_COSMETIC_DISPLAY_PRICE': False,
```

Comment:
New setting set to false here for displaying cosmetic prices (including is "Free")